2020/2020-jul.md

```diff
@@ -3,41 +3,89 @@ Development Meeting Agenda July 2020
 
 
 ## Attendees
-
-
+* Zephyyrr
+* Opl
+* Oromit
+* Popeter45
+* Tyr
+* Zoton2
 
 ## Updates from Organizers
+Not at this moment.
 
 ## Projects
 ### Tracker (Zephyyrr)
-
+Been working on the current tracker. Since last meeting I implemented a Crowd Control integration and fixed a couple minor issues.
+Packaged a proper production Docker image used to deploy both a new test server and a dedicated UKSG server.
 
 ### Speedcontrol & Layouts (Zoton2)
+They exist.
+Stuff has been done. Lots of stuff.
 
+Omnibar is still a bit crap but a lot has been rewritten.
+Looking at new design. Has been talk about bringing in a real graphics designer.
+
+Break the Record Live specific things have taken up a lot of time.
 
 ### Big Red Buttons (Popeter45)
+Physical prototype of next version. Better box, connectors instead of straigt soldering for easier disassebling.
+PoE hats for power
 
+Improved holder for Raspberry Pi.
+Since there might be some time until the next on-site event, there is no big hurry for now.
 
 ### Mobile App (Oli)
 Oli has some personal matters to attend so not attending this time.
 
-Oli is working hard on preparing the app for ESA Summer Onlines alpha test, if you are interested in joining PM him your Google Play Store email he will add you when it's available
-
+Oli is working hard on preparing the app for ESA Summer Onlines alpha test, if you are interested in joining PM him your Google Play Store email he will add you when it's available.
 
-### Discord bot (Opl)
+### Opl
+MQ Events are used.
+Can now be compiled to TypeScript.
 
+#### Music
+Looked into labels we could use.
+Found some that should be decent for both Twitch and Youtube.
 
-### Fundrasiser dashboard (Freezy)
+Made a prototype for new player with smarter playlists.
 
+#### GDQ Reminders
+Work progressing which will lead to a ESA fork.
 
 ### Video (Oromit)
+Everything works and nothing needs change.
+Will deploy new HQ router and VPN solution on Saturday.
 
+Needs more information from Social Media team about the custom tweeting tool they wanted.
 
 ### Users Tool (Cbenni)
+Not here.
+
+We appreciate his effors.
+Tyr: He is a cool guy.
+Zoton2: I am sure he still works for Stream Elements.
 
 
 ## Other questions & topics
 
+### Audio & Tech
+X32 will capture audio from four runners.
```

Comment:
Not sure what was said during the meeting but it'll technically be audio from 7, but only 2 for the main marathons (the other ones are only for BtRL)